TypeCobol/Compiler/Nodes/Data.cs

```diff
@@ -534,16 +578,29 @@ public virtual long StartPosition
                     for (int i = 0; i < Parent.Children.Count; i++)
```

Comment:
I understand what you do, but can you still add a comment about the purpose of this loop?